islandora_pdf.info.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Islandora PDF'
+description: 'A default Islandora Repository module to handle PDF files'
+dependencies:
+  - :islandora
+  - :imagemagick
+package: 'Islandora Solution Packs'
+core: 8.x
+stylesheets:
```

Comment:
Should... this stuff be moved over to an `islandora_pdf.libraries.yml`, and whatever done to make it load in the applicable spots?... It